docs/cli.md

```diff
@@ -80,186 +356,217 @@ $ oc get pods -s template=production
 
 ```
-$ oc describe service frontend
+$ oc volume dc/registry --add --mount-path=/opt
 ```
 
-oc label
-----------------
+### oc label
 
-This command adds labels to a provided resource. 
+This command adds labels to a provided resource.
 It can also overwrite the existing labels by using the `--overwrite` flag.
 
-#### Examples
-
 ```
 $ oc label service frontend foo=bar
 ```
 
-oc stop
-----------------
-
-This command gracefully shuts down a resource by id or filename. 
-It attempts to shut down and delete a resource that supports graceful termination. 
-If the resource is resizable, it will be resized to 0 before deletion.
+### oc expose
 
-#### Examples
+Expose containers internally as services or externally via routes. There is also
+the ability to expose a deployment config, replication controller, service,
+or pod as a new service on a specified port. If no labels are specified,
+the new object will re-use the labels from the object it exposes.
 
+```bash
+# Expose a service as a route
+$ oc expose service frontend
+# Expose a deployment config as a service and use the specified port and name
+$ oc expose dc ruby-hello-world --port=8080 --name=myservice --generator=service/v1
 ```
-$ oc stop service frontend
-```
-
-oc namespace
------------------
 
-This command sets the default namespace used for all `oc` commands.
+### oc stop
 
-#### Examples
+This command gracefully shuts down a resource by id or filename.
+It attempts to shut down and delete a resource that supports graceful termination.
+If the resource is resizable, it will be resized to 0 before deletion.
 
 ```
-$ oc namespace myuser
+$ oc stop service frontend
 ```
 
-oc logs
-------------
+### oc delete
 
-This command dumps the logs from a given Pod container. You can list the
-containers from a Pod using the following command:
+This command deletes a specified resource.
 
 ```
-$ oc get -o yaml pod POD_ID
+$ oc delete -f pod.json
+$ oc delete pod 1234-56-7890-234234-456456
 ```
 
-#### Examples
+## Troubleshooting and Debugging Commands
+
+### oc logs
+
+This command dumps the logs from a given Pod container.
+Use `oc get pod <pod-id>` to list the containers from a Pod.
 
 ```
 $ oc logs frontend-pod mysql-container
 ```
 
-oc expose
-------------
+### oc exec
 
-Expose containers internally as services or externally via routes. There is also
-the ability to expose a deployment config, replication controller, service, 
-or pod as a new service on a specified port. If no labels are specified, 
-the new object will re-use the labels from the object it exposes.
+This is synonymous with the [`kubectl exec`](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl_exec.md) command.
 
+### oc port-forward
 
-#### Examples
+This is synonymous with the [`kubectl port-forward`](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl_port-forward.md) command.
```

Comment:
get, create, describe and other commands are also coming straight from Kubernetes but we give a description about them instead of pointing to kubectl doc. So to be consistent we either have to redirect all wrapped commands to Kube docs or describe them. I am in favor of the later option. See the descriptions of every wrapped command in [here](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/wrappers.go).